proxy/manager.go

```diff
@@ -23,6 +23,7 @@ func (proxy *Proxy) manage(ctx context.Context, w http.ResponseWriter, req *http
 
 	// Copy headers from original request to proxy request
 	proxyReq.Header = req.Header
+	proxyReq.Host = req.Host
```

Comment:
Comment above is very slightly misleading, the `Host` is not part of the headers. Maybe just change the comment to `Copy headers and host from orig…`?